examples/ipsec-acc.p4

```diff
@@ -133,9 +133,7 @@ parser MainParserImpl(
     state start {
         main_meta.sa_index = 1; // just for exmaple, used for encrypt
 
-        // TODO: can't find  better indication of recirc in the existing pna.p4
-        // This should be a field in istd or and extern
-        transition select(istd.loopedback) {
+        transition select(istd.recirculated) {
```

Comment:
@pbhide See if this is closer to what you were hoping for, and justifies deleting the TODO comment you had.